packages/text-input/src/TextInput.tsx

```diff
@@ -75,9 +90,8 @@ export const textInputStyles = ({
 		borderStyle: 'solid',
 		borderColor: boxPalette.borderInput,
 		borderRadius: tokens.borderRadius,
-		cursor: 'pointer',
```

Comment:
Why are we dropping this?

Reply:
This was my mistake, this brings the component more inline with GOLD.